lighthouse-cli/bin.ts

```diff
@@ -114,6 +116,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', 25)
```

Comment:
we double default disableCpuThrottling as well and the extra yargs hints to users won't hurt IMO